Doc/library/fileinput.rst

```diff
@@ -148,8 +148,8 @@ available for subclassing as well:
    The sequence must be accessed in strictly sequential order; random access
    and :meth:`~io.TextIOBase.readline` cannot be mixed.
 
-   With *mode* you can specify which file mode will be passed to :func:`open`. It
-   must be one of ``'r'``, ``'rU'``, ``'U'`` and ``'rb'``.
+   With *mode* you can specify which file mode will be passed to :func:`open`.
+   It must be ``'r'`` or ``'rb'``.
```

Comment:
Can the error message in `fileinput` also be modified? 

https://github.com/python/cpython/blob/59543347d12a7717235f941e7fd363d4df92984a/Lib/fileinput.py#L214

Reply:
done